synapse/models/inet.py

```diff
@@ -240,6 +246,8 @@ def repr(self, valu):
         return ipv4str(valu)
 
     def parse(self, text, oldval=None):
+        # deal with "defanged" ipv4
+        text = text.replace('[.]','.')
```

Comment:
any reason not use castInetDeFang ? if we ever want to support >1 way of "defanging" , etc.